DigitalLearningSolutions.Data/DataServices/RequestSupportTicketDataService.cs

```diff
@@ -0,0 +1,43 @@
+﻿
+namespace DigitalLearningSolutions.Data.DataServices
+{
+    using Dapper;
+    using DigitalLearningSolutions.Data.Models.Support;
+    using System.Collections.Generic;
+    using System.Data;
+    public interface IRequestSupportTicketDataService
+    {
+        IEnumerable<RequestType> GetRequestTypes();
+        string? GetUserCentreEmail(int userId, int centreId);
+
+    }
+    public class RequestSupportTicketDataService : IRequestSupportTicketDataService
+    {
+        private readonly IDbConnection connection;
+
+        public RequestSupportTicketDataService(IDbConnection connection)
+        {
+            this.connection = connection;
+        }
+        public IEnumerable<RequestType> GetRequestTypes()
+        {
+            return connection.Query<RequestType>(@$"SELECT TicketTypeId as ID,TypePrompt AS RequestTypes,FreshdeskTicketType AS FreshdeskRequestTypes FROM [dbo].[TicketTypes]  order by TypePrompt");
+        }
+        public string? GetUserCentreEmail(int userId, int centreId)
+        {
+            //return connection.QuerySingleOrDefault<string>(
```

Comment:
Do we have any purpose for this commented query?

Reply:
Hi @rshrirohit , I had a doubt on this and added in ticket, @kevwhitt-hee  please have a look. 
![image](https://github.com/TechnologyEnhancedLearning/DLSV2/assets/120369940/487dac15-20bb-43a4-b7a5-5e86b711eb59)

Left the query commented to use if required.

Reply:
@rshrirohit  Regarding 6th point: **_Validation on file size is not working properly_**, this is a general issue . I could see same issue is in Delegate bulk upload page. If we try to upload huge file, it breaks the page irrespective of file format. I think we required a general fix for this.